cee_syslog_handler/__init__.py

```diff
@@ -48,16 +48,9 @@ def get_full_message(exc_info, message):
 
 
 #see http://github.com/hoffmann/graypy/blob/master/graypy/handler.py
-def make_message_dict(record, debugging_fields, extra_fields, fqdn, localname, facility):
-    if fqdn:
-        host = socket.getfqdn()
-    elif localname:
-        host = localname
-    else:
-        host = socket.gethostname()
+def make_message_dict(record, debugging_fields, extra_fields, facility):
     message_dict = {
-        'version': "1.0",
```

Comment:
Do we know _why_ this was there?

Reply:
[The cee_syslog handler is 90% copy past from `graypy` that logs in GELF format](https://github.com/hoffmann/graypy/blob/master/graypy/handler.py#L90). GELF requires a version string but CEE does not. I believe this was now just unnecessary copy paste.

Reply:
👍